src/com/TTT/ComputerPlayer.java

```diff
@@ -1,10 +1,24 @@
 package com.TTT;
 
 public class ComputerPlayer implements Player {
+
+    private GameUI gameUI;
+
+    public ComputerPlayer(GameUI gameUI) {
+        this.gameUI = gameUI;
+    }
+
     @Override
     public Board doTurn(Board board, PlayerMarkers marker) {
+        gameUI.clearScreen();
+        gameUI.printBoard(board);
+        gameUI.printPlayerTurn(marker);
+
         MiniMax miniMax = new MiniMax();
         board.placeMarker(miniMax.nextMove(board, marker), marker.symbol());
+
+        gameUI.pause();
+        gameUI.announceWinner(board, marker.symbol());
```

Comment:
There is a lot of feature envy here, I haven't decided how to fix it yet. One possibility was making a method to call methods on lines 13-15, but I decided it just hides the problem.